### DISCUSSION TITLE
Add the ability to create playlists in Qobuz provider

### DISCUSSION THREAD

**Discussion by madbrain76** [music-assistant/support#2790] [Feature requests and ideas]
Currently, Qobuz playslists will sync 2-way only if they are created on the Qobuz side. On the MA side, they have to be created in the built-in provider, and they don't sync to Qobuz.

Other streaming music providers reportedly allow creating playlists within MA. I'm just asking for the ability to do the same with Qobuz.

**Comment by OzGav** [comment]
This functionality will be available in 2.7.0 beta 7 or later

**Reply by madbrain76** [reply]
Thank you ! This works great !

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
